src/views/how-do-you-want-to-pay.njk

```diff
@@ -0,0 +1,62 @@
+{% extends 'layout.njk' %}
+
+{% from 'govuk/components/back-link/macro.njk'     import govukBackLink %}
+{% from 'govuk/components/error-summary/macro.njk' import govukErrorSummary %}
+{% from "govuk/components/radios/macro.njk"        import govukRadios %}
+{% from 'govuk/components/button/macro.njk'        import govukButton %}
+
+{% set title = "How do you want to pay for the application?- " + pageTitleSuffix %}
+
+{% block content %}
+  {{
+    govukBackLink({
+      text: 'Back',
+      href: Paths.HOW_DO_YOU_WANT_TO_PAY
+    })
+  }}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {% include 'components/errors/error-summary.njk' %}
+
+      <form method="post">
+        {{
+          govukRadios({
+            idPrefix: "select-payment-type",
+            name: "paymentType",
+            fieldset: {
+              legend: {
+                text: "How do you want to pay for the application?",
+                isPageHeading: true,
+                classes: "govuk-fieldset__legend--l"
+              }
+            },
+            errorMessage: {
+              text: errors.paymentType
+            } if errors.paymentType,
+            items: [
```

Comment:
I think that one of the radio buttons should be checked depending on what option the user selected upon navigating back from the Pay by Account page. Be good to confirm with Jayti.